.github/workflows/rws-submissions-checks.yml

```diff
@@ -1,63 +0,0 @@
-# Copyright 2022 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-name: RWS submission checks
-on: pull_request_target
-jobs:
-  PR-Actions:
-    runs-on: ubuntu-latest
-    outputs:
-      output1: ${{ steps.read_results.outputs.contents }}
-    steps:
-      - name: Get the PR version of the files
-        uses: actions/checkout@v2
-        with:
-          ref: "refs/pull/${{ github.event.number }}/merge"
```

Comment:
So this was intentionally running the main branch's validations against the PR, so that a malicious PR can't simply delete all the validations and bypass all requirements that way. Can we still do something like that, or are you saying that's unnecessary?

(I'm assuming that `actions/checkout` checks out the PR by default, although I don't see that mentioned in the README.)

Reply:
It's unneeded because the manual reviewer should check the files modified and ensure python wasn't touched. Checking out the main branch and using it doesn't prevent this, as it would allow one to attack in two PRs (one to modify python and a follow-up to update JSON).

Reply:
AFAIK it's an anti-pattern for actions to checkout main and use it, the one exception might be checking out a published package to use it in some way.

The important protection here is the GitHub action triggering on pull_request_target so that it's the main branches action code that runs.

Reply:
Fair, I'm just trying to avoid leaning on code review for security guarantees. But I suppose at a certain point that's inevitable.